vcpkg.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
If we have a root vcpkg.json, do we still need the vcpkg.json for cata lib and object creator, slash does this interfere with those in any way? Does it have to be located *here* or can it live in build-scripts or something?

Reply:
It can live where VCPKG_MANIFEST_DIR points to.